src/components/topCategories/TopCategories.module.css

```diff
@@ -0,0 +1,9 @@
+.categories{
+    text-decoration: none;
+    color: #13101E;
```

Comment:
please move the hex colors to theme, or have it in a constant then use it from there.
it helps you in the future when you want to change any color, you will not go through the whole project and fix it everywhere. you need to change it from one place only.

Reply:
done